opentelemetry/src/metrics/instruments/counter.rs

```diff
@@ -31,6 +33,8 @@ impl<T> Counter<T> {
 }
 
 /// An async instrument that records increasing values.
+///
+/// [`ObservableCounter`] can be cloned to create multiple handles to the same instrument. Avoid creating duplicate [`ObservableCounter`]s for the same instrument.
```

Comment:
given callbacks are already specified at creation time, is there a use case for cloning Observable instruments today?

Reply:
Good catch 😄

I'll remove this from the asynchronous instruments.